google/cloud/BUILD.bazel

```diff
@@ -211,6 +211,7 @@ cc_library(
         "@com_github_nlohmann_json//:nlohmann_json",
         "@com_google_absl//absl/functional:function_ref",
         "@com_google_absl//absl/types:span",
+        "@com_google_protobuf//:protobuf",
```

Comment:
This will make `google_cloud_cpp_rest_internal` depend on protobuf, which in turn will make `google_cloud_cpp_storage` depend on protobuf, which we need to avoid (adding dependencies to customers *only* compiling `google_cloud_cpp_storage` is a breaking change).